src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/image-list/image-list.component.html

```diff
@@ -8,7 +10,9 @@
               (fetchData)="refresh()">
     </cd-table>
   </tab>
-  <tab heading="Syncing"
+  <tab cdTabStateful
+       stateId="syncing"
```

Comment:
And how about:
 ```suggestion
  <tab cdTabStateful="syncing"
```
more clean and still the dev will know exactly what it does

Reply:
I stick to my design because `cdTabStateful` is only set on `tab` elements. But to get it fully working, the `tabset` element also needs an attribute for identification. Thus using one and the same attribute for both elements is the better and cleaner approach IMO.

Reply:
ok, I might extend this directive for other purpose in the future, so leaving it might be better.